src/Kodify/BlogBundle/Resources/config/routing.yml

```diff
@@ -1,5 +1,8 @@
 home:
     path:     /
+    defaults: { _controller: KodifyBlogBundle:Posts:index, sort: 1 }
+sorted_home:
+    path:     /sort/{sort}
```

Comment:
Yes. I know it's the same result. But apart from SEO concerns (duplicated content) i don't see any problem about it.

A quick solution could be to put sort parameter as a query parameter on home route. URL could be something like:

`http://blog.foo/?sort=1` or `http://blog.foo/?sort=2`